homeassistant/components/schlage/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: This integration does not have any actions.
+  common-modules: done
+  docs-high-level-description: done
```

Comment:
I am missing a link to the manufacturer. Also, we don't really explicitly explain what Schlage is